api/api.go

```diff
@@ -241,6 +242,12 @@ type Cluster struct {
 
 	// TunnelConfig for the cluster [key, cert, endpoint]
 	TunnelConfig TunnelConfig
+
+	// ClusterDomainsActiveMap is a map of cluster domains to a boolean value
+	// indicating which domains are active or inactive.
+	// All the nodes from domains which are inactive do not
+	// participate in quorum decisions
+	ClusterDomainsActiveMap types.ClusterDomainsActiveMap
```

Comment:
nit: suggest to use an enum for state, it is more explicit and allows expansion in future. (state = active, state = inactive).

Reply:
Added an enum ClusterDomainState in gossip.